### PR TITLE
Feature/assignment13

### PR DESCRIPTION
**Pull request by dheerajsrinath**
Create a Parent class which has a String surname = “TYSON”, And create Child class having String  firstName = “MIKE”.
Create a Test Class. In that create a Child class object and call getSurname and getFirstName. 
Note - Use inheritance concept

### CODE REVIEW

File: src/test/java/com/assignmentTest/Assignment13Test.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import org.slf4j.LoggerFactory;
+
+public class Assignment13Test {
+    private Logger logger = LoggerFactory.getLogger(this.getClass());
+
+    @Test
+
+    public void testParentChildInheritance() {
+        Child child = new Child();
+        logger.info(child.getFirstName() + " " + child.getSurname());
```
Comment (PranavHanwante) [inline src/test/java/com/assignmentTest/Assignment13Test.java]: add intent

File: src/test/java/com/assignmentTest/Assignment13Test.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class Assignment13Test {
+    private Logger logger = LoggerFactory.getLogger(this.getClass());
+
+    @Test
+
```
Comment (PranavHanwante) [inline src/test/java/com/assignmentTest/Assignment13Test.java]: remove add intent

File: src/main/java/com/assignment/Child.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package com.assignment;
+
+class Parent {
+    public String getSurname() {
```
Comment (PranavHanwante) [inline src/main/java/com/assignment/Child.java]: add intent

File: src/main/java/com/assignment/Child.java [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+        return "Tyson";
+    }
+}
+
+public class Child extends Parent {
+    public String getFirstName() {
```
Comment (PranavHanwante) [inline src/main/java/com/assignment/Child.java]: add intent

**Review by PranavHanwante — APPROVED** [review]

**Review by akhil7000 — APPROVED** [review]

**Pull request merged**